AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -18,6 +18,7 @@ amp-reddit-com.cdn.ampproject.org##.AppSelectorModal__body
 reddit.com#$#.XPromoPopup { display: none!important; }
 reddit.com#$#body.scroll-disabled { overflow: visible!important; position: static!important; }
 !
+pikicast.com##.modal
```

Comment:
Previously, this blocked the feed on the main page, now useful elements.
Try `||pikicast.com/js/view/layer/asset/layer.store.js`

Reply:
Okay. 
https://github.com/AdguardTeam/AdguardFilters/pull/82955/commits/c1c2fab53e9e58acd1f56c89fe0c602dcc4fc07a